artifactory/commands/transferfiles/filesdiff.go

```diff
@@ -200,15 +211,16 @@ func convertResultsToFileRepresentation(results []servicesUtils.ResultItem) (fil
 	return
 }
 
-func (f *filesDiffPhase) getTimeFrameFilesDiff(repoKey, fromTimestamp, toTimestamp string) (result *servicesUtils.AqlSearchResult, err error) {
-	query := generateDiffAqlQuery(repoKey, fromTimestamp, toTimestamp)
+func (f *filesDiffPhase) getTimeFrameFilesDiff(repoKey, fromTimestamp, toTimestamp string, paginationOffset int) (result *servicesUtils.AqlSearchResult, err error) {
+	query := generateDiffAqlQuery(repoKey, fromTimestamp, toTimestamp, paginationOffset)
 	return runAql(f.srcRtDetails, query)
 }
 
-func generateDiffAqlQuery(repoKey, fromTimestamp, toTimestamp string) string {
-	items := fmt.Sprintf(`items.find({"type":"file","modified":{"$gte":"%s"},"modified":{"$lt":"%s"},"$or":[{"$and":[{"repo":"%s","path":{"$match":"*"},"name":{"$match":"*"}}]}]})`, fromTimestamp, toTimestamp, repoKey)
-	items += `.include("repo","path","name")`
-	return items
+func generateDiffAqlQuery(repoKey, fromTimestamp, toTimestamp string, paginationOffset int) string {
+	query := fmt.Sprintf(`items.find({"type":"file","modified":{"$gte":"%s"},"modified":{"$lt":"%s"},"$or":[{"$and":[{"repo":"%s","path":{"$match":"*"},"name":{"$match":"*"}}]}]})`, fromTimestamp, toTimestamp, repoKey)
```

Comment:
```suggestion
	query := fmt.Sprintf(`items.find({"$and":[{"modified":{"$gte":"%s"}},{"modified":{"$lt":"%s"}},{"repo":"%s","path":{"$match":"*"},"name":{"$match":"*"}}]})`, fromTimestamp, toTimestamp, repoKey)
```
After reformatting the AQL, I got this JSON:
```json
{
  "type": "file",
  "modified": { "$gte": "%s" },
  "modified": { "$lt": "%s" },
  "$or": [
    {
      "$and": [
        { "repo": "%s", "path": { "$match": "*" }, "name": { "$match": "*" } }
      ]
    }
  ]
}
```
I have 2 comments:
1. Modified is duplicate and should be inside "and".
2. Redundant "or" clause.
3. The default type is already "file".

I suggest simplifying into this:
```json
{
  "$and": [
    { "modified": { "$gte": "%s" } },
    { "modified": { "$lt": "%s" } },
    { "repo": "%s", "path": { "$match": "*" }, "name": { "$match": "*" } }
  ]
}
```